lib/migration.js

```diff
@@ -636,8 +636,9 @@ function mixinMigration(PostgreSQL) {
       // verify that the other model in the same DB
       if (this._models[fkEntityName]) {
         return 'CONSTRAINT ' + this.escapeName(fk.name) + ' ' +
-          'FOREIGN KEY (' + this.escapeName(fk.foreignKey) + ') ' +
-          'REFERENCES ' + this.tableEscaped(fkEntityName) + '(' + fk.entityKey + ')';
+        'FOREIGN KEY (' + this.escapeName(fk.foreignKey) + ') ' +
+        'REFERENCES ' + this.tableEscaped(fkEntityName) + '(' + fk.entityKey + ')' +
+        (fk.onDelete && fk.onDelete.toUpperCase() === 'CASCADE' ? ' ON DELETE CASCADE' : '');
```

Comment:
We need to notify the user when they are using unsupported `onDelete` configuration. I see two options:

1. Log a warning via `console.warn`, continue the migration using default `ON DELETE` behavior.
2. Throw an error to abort auto-migration, force the user to pick a supported `ON DELETE` behavior.

Personally, I prefer tighter checks to avoid the situation where the user accidentally misses a warning and then complains to us about `onDelete` flag not working, therefore I would choose the second option (throw an error).

Reply:
I agree, personally I prefer to throw an error as it will force user to fix the issue, ensure that nothing is inconsistent and have a low probability to break existing code base since it's a new feature.

Reply:
I think we are in agreement then :)